src/main/java/com/aws/greengrass/clientdevices/auth/ClientDevicesAuthService.java

```diff
@@ -137,27 +142,36 @@ private int getValidCloudCallQueueSize(Topics topics) {
         return newSize;
     }
 
-
     @Override
     protected void install() throws InterruptedException {
         super.install();
         registerEventListeners();
-        subscribeToConfigChanges();
+        onConfigurationChanged();
+        registerSubscribers();
```

Comment:
Does this work? There won't be subscribers for the config change event

Reply:
Yes, I know it seems fishy but that is how was working before. Before starting the listeners we need to call this so that it boots up the configuration with some initial values. If you check what we had before the ` onConfigurationChanged()` was before inside of `subscribeToConfigChanges` which got renamed to ` registerSubscribers` and was being called before the call to ` config.lookupTopics(CONFIGURATION_CONFIG_KEY).subscribe(this::configChangeHandler);`